deployments.json

```
@@ -942,5 +942,15 @@
     "name": "Immutable zkEVM Testnet",
     "chainId": 13473,
     "url": "https://explorer.testnet.immutable.com/address/0xcA11bde05977b3631167028862bE2a173976CA11"
+  },
+  {
+    "name": "RSS3 VSL Mainnet",
+    "chainId": 12553,
+    "url": "https://scan.mainnet.rss3.io/address/0xcA11bde05977b3631167028862bE2a173976CA11"
```

This gives me a 404, I think this is the correct URL?
```suggestion
    "url": "https://scan.rss3.io/address/0xcA11bde05977b3631167028862bE2a173976CA11"
```

---

fixed.